### ISSUE TITLE
Broken network widget in dashboard (3.8-12.13)

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#1289]
The Network dashboard widget is now non functional in 3.8-12.13 with repeated errors of:-
"storageadmin.js:18826 Uncaught TypeError: Cannot read property 'push' of undefined"
```
18825   _.each(data.results, function(d) {
18826      _this.dataBuffers[d.device].push(d);
18827    });
```

**Comment by phillxnet** [comment]
Linked to possible duplicate #1302.

**Comment by MFlyer** [comment]
To @schakrava : issue to be closed like 1302 via PR #1323  :wink:

**Comment by phillxnet** [comment]
@MFlyer Sorry forgot to close this issue, it was hanging around as a duplicate origin and although older than it's duplicate it got less attention. Thanks for diligently labelling your commits as relevant here. 

Closing as fix committed by @MFlyer and merged by @schakrava via pr #1323
